src/hooks/screen/useChatScreenHook.ts

```diff
@@ -37,9 +39,12 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
   const [selectedMessageId, setSelectedMessageId] = React.useState<string | null>(null);
   const {selectedChannel, goBackFromChatScreen, goToChatInfoScreen} = useChatUtilsHook();
   const {replyPreview, clearReplyPreview} = useMessageHook();
+  const {anonProfileId} = useUserAuthHook();
   const flatListRef = React.useRef<FlatList>(null);
   const [loadingChat, setLoadingChat] = React.useState(true);
   const [chats, setChats] = React.useState<ChatSchema[]>([]);
+  const [selfAnonUserInfo, setSelfAnonUserInfo] = React.useState<any>(null);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new state variable `selfAnonUserInfo` has been introduced. Ensure that this state variable is properly initialized and updated throughout the component lifecycle to avoid potential issues with undefined or stale state.
```diff
+ const [selfAnonUserInfo, setSelfAnonUserInfo] = React.useState<any>(null);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->